projects/novo-elements/src/elements/data-table/cell-headers/data-table-header-cell.component.ts

```diff
@@ -193,6 +195,9 @@ export class NovoDataTableCellHeader<T> implements IDataTableSortFilter, OnInit,
   @Input()
   allowMultipleFilters: boolean = false;
 
+  @Input()
+  dataFeatureId: string;
+
```

Comment:
wondering if you can directly use `column.id` instead of passing it through this input.. is that value available to the template?

Reply:
cool, i'll give that a shot